RELEASE/scripts/autoscend/auto_util.ash

```diff
@@ -2726,6 +2726,86 @@ int doNumberology(string goal, boolean doIt, string option)
 	return -1;
 }
 
+boolean candyEggDeviler()
+{
+	if(!(item_amount($item[Candy Egg Deviler]) > 0 || storage_amount($item[Candy Egg Deviler]) > 0))
+	{
+		//do we have a Candy Egg Deviler?
+		return false;
+	}
+	if(!(get_property("_candyEggsDeviled").to_int() < 3))
+	{
+		//already generated our 3 deviled candy eggs today
+		return false;
+	}
+
+	if(storage_amount($item[Candy Egg Deviler]) > 0)
+	{
+		pullXWhenHaveY($item[Candy Egg Deviler], 1, 0);
+	}
+
+	//Below is modified from the synthesis code
+	int maxprice = 2500;
+	if(get_property("auto_maxCandyPrice").to_int() != 0)
+	{
+		maxprice = get_property("auto_maxCandyPrice").to_int();
+	}
+
+	item[int] candyList;
+	foreach it in $items[]
+	{
+		if(it.candy && (item_amount(it) > 0) && (auto_mall_price(it) <= maxprice) && it.tradeable)
```

Comment:
this will miss any untradable candy we happen to have (most likely Comet Pops from the Model Train Set, also black candy heart / explosive chewing gum)

I think these should be valued over mall-sellable candy